_security/access-control/document-level-security.md

```diff
@@ -9,7 +9,8 @@ redirect_from:
 ---
 
 # Document-level security
-Document-level security lets you restrict a role to a subset of documents in an index. The easiest way to get started with document- and field-level security is to open OpenSearch Dashboards and choose **Security**. Then choose **Roles**, create a new role, and review the **Index Permissions** section, shown in the following image.
+
+Document-level security (DLS) controls which documents a role can retrieve when reading from an index. It applies only to read operations, such as search and get, and does not restrict indexing, updating, or deleting documents. If a role has write or delete permissions on an index, it can still modify or remove documents that are hidden by DLS. The write permissions are governed solely by index permissions and action groups. To get started with document level security, open OpenSearch Dashboards and choose **Security**. Then choose **Roles**, create a new role, and review the **Index permissions** section, shown in the following image.
```

Comment:
```suggestion
Document-level security (DLS) controls which documents a role can retrieve during read operations, such as search and get. It does not restrict write operations. If a role has permissions to index, update, or delete documents on an index, it can still modify or remove documents that are hidden by DLS. Write behavior is determined solely by index permissions and action groups.

To get started with DLS, open OpenSearch Dashboards and choose **Security**. Then select **Roles**, create a new role, and review the **Index permissions** section shown in the following image.
```